apps/ledger-live-mobile/src/screens/SendFunds/03a-AmountCoin.tsx

```diff
@@ -159,7 +160,10 @@ export default function SendAmountCoin({ navigation, route }: Props) {
                 onChange={onChange}
                 value={amount}
                 error={
-                  amount.eq(0) && (bridgePending || !transaction.useAllAmount)
+                  status.errors.dustLimit
+                    ? status.errors.dustLimit
+                    : amount.eq(0) &&
+                      (bridgePending || !transaction.useAllAmount)
                     ? null
                     : status.errors.amount
```

Comment:
Minor optim, but maybe a `const error = useMemo(...)`?

Reply:
Mmm I don't think this is the intented usage of `useMemo` as it's mostly usefull to cache heavy calculation at every rerender (e.g when no props changed, API calls). I'm more in favor of not overusing hooks as a solution to core performance problems (too many rerenders for exemple)

Reply:
Some argument about memoizing everything https://attardi.org/why-we-memo-all-the-things/ ^^
But really nonimportant, nbd.